WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -129,7 +129,6 @@
                     <Button.Content>
                         <FontIcon
                             x:Name="CopyLinkButtonIcon"
```

Comment:
This would need `FontSize="16"` as well?

Reply:
"need" seems to be a difficult word here. If it defaults to 16, I don't think it **needs** to. Same discussion for the FontFamily.

Reply:
As mentioned above, `FontIcon` defaults to 20. That's what's happening now:

<img width="123" alt="image" src="https://user-images.githubusercontent.com/9866362/231168865-6334dce3-2623-44c9-9782-f243becf6bfb.png">

When `FontSize` is set to 16 it renders the correct size (with the offset as mentioned earlier):
<img width="133" alt="image" src="https://user-images.githubusercontent.com/9866362/231169243-3eba1113-e5d6-43ff-b85a-f1feec2ef809.png">

So we'd need to explicitly set it to make sure all icons render the same size.